nalej/web
# Eslint, typescript-eslint and prettier instalation

#### What does this PR do?
- Upgrades to node version [10.17.0](https://github.com/nodejs/docker-node/blob/c6bc44e84afcdb81d9749b7b034c60e916a519ad/10/alpine3.9/Dockerfile)
- Adds rules and installs: 
    1. [Prettier](https://prettier.io)
    2. [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint)
    3. [Eslint](https://eslint.org/)

#### Where should the reviewer start?
Checking the new files .pretierrc.json .eslintrc.json

#### What is missing?
The implementation of the desired rules will be added in  [NP-2065](https://nalej.atlassian.net/browse/NP-2065) and  [NP-2079](https://nalej.atlassian.net/browse/NP-2079)

#### How should this be manually tested?
Run `npm install` and check that everything runs ok.
If wanted, go to packaje.json and run any rule such as:
- "eslint"
- "eslint-check"
- "eslint-fix"
- "format-all"
- "format-check"
- "format-fix"
- "prettier"

#### What are the relevant tickets?

- [NP-2084](https://nalej.atlassian.net/browse/NP-2084)
